modulo_1/Página Guilherme Martins/git_repo.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Minha página de teste 2</title>
+	<link href="styles2.css" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=League+Spartan:wght@200&display=swap" rel="stylesheet">
+  </head>
+  <body>
+	<h1> Pesquisa de usuário Github</h1>
+	<img src="search.png" alt="minha página de teste">
```

Comment:
Neste caspo temos um ótimo SEO aplicado. Ainda sim podemos aprimorar para utilizar o alt e o atributo title.

https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/